imports/plugins/included/default-theme/client/styles/products/variantForm.less

```diff
@@ -1,32 +1,4 @@
 // == main variant edit form
-.variant-edit-form {
-  cursor: default;
-  border-radius: 0 0 6px 6px;
-  white-space:nowrap;
-  overflow: hidden;
-  font-size: 12px;
-  .well {
-    margin-bottom: 6px;
-  }
-
-  label {
-    margin-bottom: 0px;
-  }
-  .variant-form-buttons {
-    padding-top: 10px;
-  }
-}
-
-.variant-options {
-  .form-group {
-
-  }
-  .checkbox,.radio {
-    margin-top: 0;
-    margin-bottom: 0;
-  }
-}
-
 .admin-controls-content .panel-default.variant-option-card > .panel-heading {
```

Comment:
@loan-laux where do you see these values being used? I don't see them being used anywhere, we could probably get rid of this entire file.

Reply:
`admin-controls-content` is used in the `ActionView` component. Is that still being used though? I tried to dive into it but it's not really clear. Happy to delete if you can confirm. The less files the better!

Reply:
Although the `ActionView` component still seems to be used (it's rendered directly in the `App` component and seems to be a core component of the routing system), I removed the CSS rules and no changes seemed to have happened. I guess a refactoring of this whole `ActionView` thing is in order, but that would be a separate and likely massive PR.